account_invoice_import_invoice2data/README.rst

```diff
@@ -92,6 +92,20 @@ Refer to the usage section of the module *account_invoice_import*.
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/226/8.0
 
+Templates for invoice2data
+==========================
+
+This module was written to provide database storage and a semmi-convenient editor for invoice2data templates. By itself, this module does nothing, you need to install some other module that uses invoice2data templates.
```

Comment:
Please update this stanza to the way the functionality was merged into the main module. Also mention the `invoice2data_exclude_built_in_templates config` option

Reply:
This is already in the README:
```invoice2data_templates_dir = /opt/invoice2data_local_templates
  invoice2data_exclude_built_in_templates = True```